linter/whitelists/de.txt

```diff
@@ -61,6 +61,7 @@ Collapse
 Columns
 Community
 Composer
+Config
```

Comment:
You can add this to the `default` list, then you don‘t have to add it twice :)

Reply:
Done. Note: That's the way I did it initially, but then I read the README more carefully before committing, and it referenced proper names only, which is, in fact, all there was in the default.txt at the time.